mopidy/core/history.py

```diff
@@ -57,3 +57,27 @@ def get_history(self):
         :rtype: list of (timestamp, :class:`mopidy.models.Ref`) tuples
         """
         return copy.copy(self._history)
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
+
+        # 500 tracks a 3 minutes -> 24 hours history
+        count_max = 500
+        count = 1
+        history_list = []
+        for timestamp, track in self._history:
+            history_list.append(HistoryTrack(
+                                timestamp=timestamp, track=track))
+            count += 1
+            if count_max < count:
+                logger.info('Limit history to %s tracks.', count_max)
```

Comment:
s/Limit/Limiting/ to be clear that we're limiting it, not that the user should go limit it somehow.